src/common/interfaces/statics.ts

```diff
@@ -35,13 +36,6 @@ export interface Bank {
   config: string;
 }
 
-export interface DateeFormat {
-  id: string;
-  format: string;
-  format_moment: string;
-  format_dart: string;
-}
-
```

Comment:
Quick reference to the updated type

https://github.com/invoiceninja/ui/blob/fbb22e5363feabd1bb28154bf5111ddb951d7766/src/common/interfaces/date-format.ts#L11-L16